lib/game_stats.rb

```diff
@@ -8,4 +8,16 @@ class GameStats
   def initialize(file_path, object)
     @games = csv_data('./data/games_truncated.csv', Game)
   end
+
+  def highest_total_score
+    @games.map {|game| game.away_goals + game.home_goals}.max
```

Comment:
Ruby style guide prefers a space before and after curly brackets. E.g., `@games.map { |game| game.away_goals + game.home_goals }.max`

Reply:
Good job on saving line space by putting max at the end